docs-source/content/userguide/introduction/architecture.md

```diff
@@ -67,7 +67,7 @@ During a rolling event caused by a change to the Domain's `image` field, contain
 Within the container, the following aspects are configured by the operator:
 
 *	The `ENTRYPOINT` is configured by a script that starts up a Node Manager process, and then uses WLST to request that Node Manager start the server.  Node Manager is used to start servers so that the socket connection to the server will be available to obtain server status even when the server is unresponsive.  This is used by the liveness probe.
-* The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container.
+* The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container. The liveness probe interval and timeout can be customized using the `livenessProbe` attribute under the `serverPod` element of the domain resource. It is also possible to customize the liveness probe by using `livenessProbeCustomScript` field in the domain resource or by setting `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable. These options can be used to provide a custom script which will be invoked by the existing liveness probe script (`livenessProbe.sh`) for WebLogic Server Pods after the existing script performs its own checks. If the custom script fails with non-zero exit status, the liveness probe will fail and Kubernetes will restart the container.
```

Comment:
I wonder if we should should support both setting the env var and a domain resource attr - there's too much potential for confusion by supporting both (which takes precedence?, yada, yada, yada).  

Note that the env var approach automatically gives you the ability to customize on a fine-grained basis - since one can pick-and-choose which servers/clusters within a domain get which env vars, but, OTOH, we have historically tended to avoid supporting env vars when adding a domain resource attribute was aa practical option.   We could get the same fine-grained advantage for a domain resource attribute by scoping it into the serverPod stanza...

Reply:
I believe there are other places where we support both such as server shutdown type, timeout and shutdown ignore sessions, startup mode and dynamic config override. 

The environment variable takes precedence since domain resource attr will only set the env variable if it's not already set. IIRC, Ryan had previously suggested having both domain resource attr and env variable as that provides more flexibility. However I'm open to changing it as @rjeberhard and you will suggest. Thanks.